feature_engine/selection/drop_psi_features.py

```diff
@@ -135,10 +136,12 @@ class DropHighPSIFeatures(BaseSelector):
         test) will be switched. This is important because the PSI is not symmetric,
         i.e., PSI(a, b) != PSI(b, a)).
 
-    threshold: float, default = 0.25.
+    threshold: float, str, default = 0.25.
         The threshold to drop a feature. If the PSI for a feature is >= threshold, the
         feature will be dropped. The most common threshold values are 0.25 (large shift)
         and 0.10 (medium shift).
+        If 'auto' than threshold will be calculated based on size of basis and test 
+        dataframes and number of bins with formula proposed by B. Yurdakul.
```

Comment:
I think this 2 lines need a bit of editing for typo and grammar. Could you take a look? Also, I would add a note "See references below", so that people know where to find that formula.

Reply:
Ok, will do